### ISSUE TITLE
About The Proposal File

### ISSUE THREAD

**Issue by tianyuecao** [bradezard131/wsod#4]
I got the error `FileNotFoundError: [Errno 2] No such file or directory: 'anno/proposals_trainval.pkl'`. Could you please provide the proposal file? Or how can I get the proposal file? Thanks!

**Comment by liz6688** [comment]
Do you solve the problem? If your problem is solved, can you tell me how to generate a proposal file？

**Comment by tianyuecao** [comment]
> Do you solve the problem? If your problem is solved, can you tell me how to generate a proposal file？

yes, i have found a proposal file [here.](https://github.com/ppengtang/pcl.pytorch)
